content/en/docs/ibm/using-icr.md

```diff
@@ -28,20 +26,20 @@ kubectl get secret all-icr-io -n default -o yaml \
 | kubectl -n anonymous create -f -
 ```
 
-Once this secret is ready in your Kubeflow profile, A data scientist can use this secret to pull container images from ICR.
+Once this secret is ready in your Kubeflow profile, a data scientist can use this secret to pull container images from ICR.
 
 See details and FAQs from the official guide [Setting up an image registry](https://cloud.ibm.com/docs/containers?topic=containers-registry).
 
-### Use IAM API Key
+### Getting an IAM API Key
 
-You will need to use an IBM Cloud IAM API Key to work with ICR in below cases:
-1. has no access to the default image pull secret `all-icr-io` from the `default` namespace.
-1. need to access container images in other IBM Cloud accounts.
-1. need customized IAM policy by using a separate IAM sevice id.
+You will need an IBM Cloud IAM API Key to work with ICR if you:
+1. Have no access to the default image pull secret `all-icr-io` from the `default` namespace.
```

Comment:
Since not all Markdown may be rendered as desired, use bullet points, instead of a numerical sequence of 1-1-1 to make sure users don't see Step 1, Step1, Step 1.

> * Have no access...
> * Need to access...
> * Need customized IAM ...

I did see your comment, so this is a new suggestion.

> Thanks! I use 1,1,1 since the number doesn't have to be in number order. see https://www.markdownguide.org/basic-syntax#ordered-lists

Reply:
Basically, this is what happens otherwise, when you use ordered-lists while keeping each first number at "1":
```
1. Have no access...
1. Need to access...
1. Need customized IAM...
```

Reply:
alright, let me count :)